bundles/pom.xml

```diff
@@ -147,6 +142,7 @@
     <module>org.openhab.binding.feed</module>
     <module>org.openhab.binding.feican</module>
     <module>org.openhab.binding.fenecon</module>
+    <module>org.openhab.binding.ferroamp</module>
```

Comment:
This line should be the only change to this file. Please revert other changes to this file.

Reply:
Also see https://github.com/openhab/openhab-addons/pull/17536#discussion_r2001764396

Reply:
Hi,
this is so strange.
I do the changes in file and Commit it with only the line 145 changed, but the other changes will still be there also.
I will continue to investigate this, but if you have some thoughts about it , Please let me know.

Reply:
You can copy paste the original file from this repository and then change this line. Have seen this before. Hopefully this helps.

Reply:
Done as described:
copied openhab-addons/bundles/pom.xml
added line 145
Commit